nxviz/geometry.py

```diff
@@ -24,6 +24,30 @@ def node_theta(nodelist, node):
     return theta
 
 
+def group_theta(node_length, node_idx):
+
```

Comment:
I'm having a little bit of trouble figuring out what exactly to write for the docstring. Could you take a stab at this one please?

```python
def group_theta(node_length, node_idx):
    """
    Returns an angle corresponding to the node index.

    Intended to be used for placing node group labels at the correct spot.

    :param float node_length: ...
    :param int node_idx: ...
    :returns: ...
    """